### ISSUE TITLE
Gatttool is deprecated

### ISSUE THREAD

**Issue by Kernald** [ThomDietrich/miflora-mqtt-daemon#70]
While it's still available on most distributions, [gatttool has been officially deprecated by the BlueZ team two years ago](https://git.kernel.org/pub/scm/bluetooth/bluez.git/commit/?id=b1eb2c4cd057624312e0412f6c4be000f7fc3617). Is there any plan to migrate to something else? (I just ran into this when trying to use miflora-mqtt-daemon for the first time today.)

**Comment by Kernald** [comment]
Sorry for the noise, I just opened a similar issue where it actually belongs: https://github.com/open-homeautomation/miflora/issues/121

**Comment by ThomDietrich** [comment]
Yes definitely planned to move away from gatttool. I did not find the time for it yet and plant care during winter time (Germany) is not my highest priority :) Thanks for reaching out!

**Comment by Markkuuss** [comment]
Wouldn't it be possible to simply change "GatttoolBackend" to "BluepyBackend" in this line?

https://github.com/ThomDietrich/miflora-mqtt-daemon/blob/c4aae6ec9bbccdb58cac813e5e6cd1422e78b1dc/miflora-mqtt-daemon.py#L220

Has this ever been tried and tested?

Maybe this is also a solution for the current kernel issue https://github.com/ThomDietrich/miflora-mqtt-daemon/issues/83?

**Comment by ThomDietrich** [comment]
I did not try it. Please try and let us know :)
For a permanent change the solution might need some more testing and definitely some updates to the changes. Please contribute a PR! Thanks :)

**Comment by bigkraig** [comment]
`BluepyBackend` seems to be working fine for me.

**Comment by Markkuuss** [comment]
@bigkraig: Can you post your Linux version (output of the command "uname -a") you tested with?

**Comment by bigkraig** [comment]
```Linux pi1 4.19.66-v7+ #1253 SMP Thu Aug 15 11:49:46 BST 2019 armv7l GNU/Linux
```

**Comment by Markkuuss** [comment]
How long has your system been running stable with this configuration without restarting the system or deamon?
